server/protocols/webadmin-integration-test/pom.xml

```diff
@@ -155,5 +184,10 @@
             <artifactId>testcontainers</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.james</groupId>
+            <artifactId>james-server-jmap-draft-integration-testing</artifactId>
+            <scope>test</scope>
```

Comment:
Is this dependency still needed?

(It should not!)

Reply:
one of my attempt to run LinshareIntegrationTest on Memory James but didn't success. It should be removed